org-formation/040-budgets/_tasks.yaml

```diff
@@ -22,3 +32,5 @@ CostAndUsageReports:
     Region: !Ref primaryRegion
   Parameters:
     resourcePrefix: !Ref resourcePrefix
+    bucketName: !Sub "${resourcePrefix}-cost-reports"
+    lambdaArn: !CopyValue CostReportFunctionArn
```

Comment:
prepend `CostReportFunctionArn` with CostReportLambda StackName or/and primaryRegion.